hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTable.java

```diff
@@ -140,11 +135,14 @@ public void onEvents(final List<?> events) {
         if (TargetCreatedEvent.class.isInstance(firstEvent)) {
             onTargetCreatedEvents();
         } else if (TargetInfoUpdateEvent.class.isInstance(firstEvent)) {
-            onTargetInfoUpdateEvents((List<TargetInfoUpdateEvent>) events);
+            onTargetUpdateEvents(((List<TargetInfoUpdateEvent>) events).stream()
```

Comment:
That won't work as we need to work on the entire list on on every target but I will check if I can optimise this a bit.